pkg/services/oracle/response.go

```diff
@@ -27,8 +27,8 @@ func (o *Oracle) getResponse(reqID uint64, create bool) *incompleteTx {
 	return incTx
 }
 
-// AddResponse processes oracle response from node pub.
-// sig is response transaction signature.
+// AddResponse processes an oracle response from the node pub.
```

Comment:
It's not well-written originally. We're dealing with a particular response from the node identified by the known public key, the response is signed by the corresponding private key and is given in the parameters.

Reply:
`AddResponse handles an oracle response (transaction signature for some identified request) signed by the given key`, for example.

Reply:
Ok, thanks